### PR TITLE
update covid19 banner and tile logic with dfocvc

### PR DESCRIPTION
**Pull request by KeithLyonMastek**
Story [AB#13379](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/13379)    
Task [AB#13564](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/13564)

### CODE REVIEW

**Review by bjss-chris — COMMENTED** [review]
I'm not sure we need to add a new feature flag. Could we just update the condition that we use to display the Reduce GP Visits tile?

In `app/pages/homepage/template.njk`, could we change line 41 to this?

```javascript
{% if (config.showCovid19 === 'true' && config.showDfocvc !== 'true') %}
```

**Comment by KeithLyonMastek** [comment]
I am going to simplicity in logic and obvious what a flag is doing as it only has 1 purpose.

A nicer solution is to call a method to determine if it should be shown, like if showBanner(), However I went with the simpliest solution in keeping with the current way of using flags

**Comment by bjss-chris** [comment]
> The new solution we want is to show the tiles but not the banner

I'm a bit late to the party on this one but I think the banner issue was a bit of a red herring and was caused by an incorrect environment variable setting, so I don't think we need an additional flag to control the banner although I understand your argument.

As which tiles are displayed can be determined from the existing feature flags I don't think we need the extra flag right now. At least we don't need it to satisfy our requirements. Therefore, I think it would be preferable to stick with the simpler option that I've suggested for now.

If necessary for future requirements we can always introduce an additional flag later.

**Review by bjss-chris — COMMENTED** [review]
Changes look good.

Is there a way we can set the relevant feature flag(s) in the UI tests so we can test that the tile is/isn't displayed depending on the setting of the flag?

File: app/pages/homepage/template.njk [hunk 1]
```diff
@@ -38,7 +38,7 @@
         </li>
       {% endif %}
 
-      {% if (config.showCovid19 === 'true') and (config.showDfocvc === 'false') %}
+      {% if (config.showCovid19 === 'true') and (config.showDfocvc != 'true') %}
```
Comment (bjss-chris) [inline app/pages/homepage/template.njk:41]: Should we be using the strict inequality operator here?

**Review by bjss-chris — APPROVED** [review]

**Pull request merged**